samtools.1

```diff
@@ -1351,38 +1357,10 @@ Disable read-pair overlap detection.
 .B Output Options:
 .TP 10
 .BI "-o, --output " FILE
-Write pileup or VCF/BCF output to
+Write pileup output to
 .IR FILE ,
 rather than the default of standard output.
-
-(The same short option is used for both
```

Comment:
I fear we still need to mention this horror somehow, if only to say that you need to use `-o ./123` instead of `-o 123` due to historic overloading of the `-o` option.